go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/scheduler-plugins
 
-go 1.17
+go 1.18
```

Comment:
Any particular reason for bumping go version?

Reply:
Sorry I forgot 1.24 is not on 1.18 yet, fixed.